pontoon/base/static/js/main.js

```diff
@@ -479,18 +479,19 @@ $(function() {
 
   // Menu sort
   $('.menu .sort span').click(function (e) {
-    function val(index, el) {
-      if (index !== 2) {
-        return $(el).find('span:eq(' + index + ')').html();
+    function getString(el) {
+      return $(el).find('span:eq(' + index + ')').html();
+    }
 
-      } else {
-        if (!$(el).find('.chart').length) {
-          return 0;
-        }
-        var chart = $(el).find('.chart').data('chart'),
-            data = JSON.parse(chart.replace(/'/g, "\""));
-        return data.approved/data.total;
+    function get(el, type) {
+      if (!$(el).find('.chart').length) {
```

Comment:
You should save this to a variable and avoid calling `$(el).find()` twice. But there's an ever better way: `data()` returned `undefined` if the jQuery object it's called on is empty, so you could even just do:

```js
var chartData = $(el).find('.chart').data('chart');
if (chartData) {
    // Return
} else {
    // Fail return
}
```